README.md

```diff
@@ -79,7 +79,7 @@ v := s.Values()           // Get a []interface{}
 f := s.Fields()           // Get a []*Field
 n := s.Names()            // Get a []string
 f := s.Field(name)        // Get a *Field based on the given field name
-f, ok := s.FieldsOk(name) // Get a *Field based on the given field name
+f, ok := s.FieldOk(name) // Get a *Field based on the given field name
```

Comment:
Can you add a whitespace before the `//` comment directive, so they are all aligned?